src/access_coupler/auscom_ice_parameters.F90

```diff
@@ -63,7 +63,7 @@ module auscom_ice_parameters_mod
 !
 integer :: sfix_hours = 12  !do s mixing every sfix_hours.
 !
-logical :: redsea_gulfbay_sfix = .true.
+logical :: redsea_gulfbay_sfix = .false.
```

Comment:
This may be turned off in namelists, but the default is different

Reply:
Seems innocuous but is a real pain to have made this change, as we cannot possibly know all of the different simulations that may rely on the default value.

Reply:
If I recall, it's a very unnatural forcing.  Probably for the best if it's disabled by default.